### PR TITLE
Virtual meetings guide with appropriate gitignore

### PR DESCRIPTION
**Pull request by kimviens**
Rajouté le guide des réunions virtuelles!

### CODE REVIEW

File: src/fr/guides/virtual-meetings/index.html [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+			<p class="fn-rtn"><a href="#fn4-rf"><span class="wb-inv">Revenir à la note de bas de page </span>4</a></p>
+			<p><a href="https://www.gcpedia.gc.ca/gcwiki/images/1/14/Guide-Planifier_des_des_%C3%A9v%C3%A9nements_inclusifs_et_accessibles.pdf">Planification d’événements inclusifs et accessibles – Emploi et Développement social Canada (accessible seulement sur le réseau interne du GC)</a></p>
+		</dd>
+		<dt id="fn5-dt">Note de bas de page 5</dt>
+		<dd id="fn5" tabindex="-1">
+			<p class="fn-rtn"><a href="#fn5-rf"><span class="wb-inv">Revenir à la note de bas de page </span>5</a></p>
+			<p><a href="https://readabilityguidelines.co.uk/audiences-devices-channels/presentations/"><span lang="fr">Readability Guidelines – Content Design London (en anglais)</span></a></p>
+		</dd>
+		<dt id="fn6-dt">Note de bas de page 6</dt>
+		<dd id="fn6" tabindex="-1">
```
Comment (shawnthompson) [inline src/fr/guides/virtual-meetings/index.html:355]: remove the `tabindex="-1"` in the `dd` element

See [WET: Working Examples: Footnotes](https://wet-boew.github.io/v4.0-ci/demos/footnotes/footnotes-en.html) for more information.

**Review by shawnthompson — CHANGES_REQUESTED** [review]
Please make changes to both the English and the French documents

File: src/fr/guides/virtual-meetings/index.html [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+			</li>
+		</ul>
+	<p><strong>Choisissez une plate-forme dotée de caractéristiques d’accessibilité telles que la compatibilité avec les appareils d’assistance et une fonction d’épinglage vidéo.</strong></p>
+		<ul class="lst-spcd">
+			<li>
+				<p>Recommandation: Microsoft Teams (mais n’oubliez pas qu’aucun outil n’est totalement accessible).</p>
+			</li>
+		</ul>
+	<div class="alert alert-info">
+		<p><strong>CONSEIL :</strong> La version de bureau de Microsoft Teams prend en charge plus de caractéristiques d’accessibilité que la version de navigation.</p>
```
[the remark this answers is not in the document]
Comment (kimviens) [reply src/fr/guides/virtual-meetings/index.html:53]: Done!

File: src/fr/guides/virtual-meetings/index.html [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+	<li><p>Interprétation des langues officielles et autochtones :</p>
+		<ul class="lst-spcd">
+			<li>Téléphone: 613-996-3346</li>
+			<li>Courriel : <a href = "mailto:btinterpretationlo.tbolinterpretation@tpsgc-pwgsc.gc.ca">btinterpretationlo.tbolinterpretation@tpsgc-pwgsc.gc.ca</a></li>
+		</ul>
+	</li>
+</ul>
+<h4>Ressource supplémentaire des États-Unis :</h4>
+<ul class="lst-spcd">
+	<li><a href="https://www.section508.gov/create/accessible-meetings/">Section 508.gov</a> de la GSA (en anglais).</li>
```
[the remark this answers is not in the document]
Comment (kimviens) [reply src/fr/guides/virtual-meetings/index.html:324]: Done!

File: src/fr/guides/virtual-meetings/index.html [hunk 4]
```diff
@@ -0,0 +1,387 @@
...
+		</dd>
+		<dt id="fn4-dt">Note de bas de page 4</dt>
+		<dd id="fn4" tabindex="-1">
+			<p class="fn-rtn"><a href="#fn4-rf"><span class="wb-inv">Revenir à la note de bas de page </span>4</a></p>
+			<p><a href="https://www.gcpedia.gc.ca/gcwiki/images/1/14/Guide-Planifier_des_des_%C3%A9v%C3%A9nements_inclusifs_et_accessibles.pdf">Planification d’événements inclusifs et accessibles – Emploi et Développement social Canada (accessible seulement sur le réseau interne du GC)</a></p>
+		</dd>
+		<dt id="fn5-dt">Note de bas de page 5</dt>
+		<dd id="fn5" tabindex="-1">
+			<p class="fn-rtn"><a href="#fn5-rf"><span class="wb-inv">Revenir à la note de bas de page </span>5</a></p>
+			<p><a href="https://readabilityguidelines.co.uk/audiences-devices-channels/presentations/"><span lang="fr">Readability Guidelines – Content Design London (en anglais)</span></a></p>
```
[the remark this answers is not in the document]
Comment (kimviens) [reply src/fr/guides/virtual-meetings/index.html:352]: I meant to put lang="en" I accidentally made them all lang="en". Good catch!

File: src/fr/guides/virtual-meetings/index.html (hunk 1 above)
Comment (kimviens) [reply src/fr/guides/virtual-meetings/index.html:355]: Done!

File: src/fr/guides/virtual-meetings/index.html [hunk 5]
```diff
@@ -0,0 +1,387 @@
...
+filename: "/docs/alternate-formats/PratiquesExemplairesPourLaTenueDeRéunionsVirtuellesAccessibles.docx"
+version: 2
+size: 93
+sizeFormat: ko
+---
+
+<h2 class="mrgn-bttm-lg">Autres formats du document (à télécharger)</h2>
+<ul class="btn-toolbar list-inline" xmlns="http://www.w3.org/1999/html">
+	<li class="btn-group mrgn-bttm-md">
+		<a href="/docs/alternate-formats/PratiquesExemplairesPourLaTenueDeRéunionsVirtuellesAccessibles.docx" class="btn btn-primary"><i class="fas fa-file-word" aria-hidden="true"></i>{{title}} (Word version 2, 93 <abbr title="kilo-octet" aria-label="kilo-octet">ko</abbr>)</a>
```
[the remark this answers is not in the document]
Comment (kimviens) [reply src/fr/guides/virtual-meetings/index.html:16]: Done!

**Comment by kimviens** [comment]
Hi Shawn! I just took care of what you asked for, ready to merge!

File: src/fr/guides/virtual-meetings/index.html [hunk 6]
```diff
@@ -0,0 +1,313 @@
...
```
Comment (shawnthompson) [inline src/fr/guides/virtual-meetings/index.html:16]: Please remove `aria-label="kilo-octet"` using both `title` and `aria-label` can cause a reader to repeat the values.

**Review by shawnthompson — CHANGES_REQUESTED** [review]

**Comment by kimviens** [comment]
Hi Shawn! Aria-label is removed, good to go!

File: src/fr/guides/virtual-meetings/index.html (hunk 6 above)
Comment (kimviens) [reply src/fr/guides/virtual-meetings/index.html]: done

File: src/fr/guides/virtual-meetings/index.html [hunk 7]
```diff
@@ -0,0 +1,313 @@
...
+documents:
+fileTitle: "Pratiques exemplaires de tenue de réunions virtuelles accessibles"
+filename: "/docs/alternate-formats/PratiquesExemplairesPourLaTenueDeRéunionsVirtuellesAccessibles.docx"
+version: 2
+size: 93
+sizeFormat: ko
+---
+
+<h2 class="mrgn-bttm-lg">Autres formats du document (à télécharger)</h2>
+<ul class="btn-toolbar list-inline" xmlns="http://www.w3.org/1999/html">
```
[the remark this answers is not in the document]
Comment (kimviens) [reply src/fr/guides/virtual-meetings/index.html:14]: done!

**Comment by kimviens** [comment]
Hi Shawn, the lang attributes have been removed from product names.

**Review by shawnthompson — APPROVED** [review]

**Pull request merged**